### PR TITLE
Add and run flake8 in Travis

### PR DESCRIPTION
**Pull request by stephendonner**
Before this lands (assuming it gets approved), we'll also need https://github.com/citp/OpenWPM/pull/200 in, with a rebase.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,10 +1,12 @@
 beautifulsoup4
 dill
+flake8==3.5.0
```
Comment (englehardt) [inline requirements.txt]: Why pin to `3.5.0`?

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,2 @@
+[flake8]
+ignore = E501
```
[the remark this answers is not in the document]
Comment (englehardt) [reply setup.cfg]: I think it's fine to skip E501 for now, but let's add a comment to #116 to that effect. Do you have a count of how common it is?

It's definitely fine to not address the latter half of #116. Those can happen in future PRs.

[the remark this answers is not in the document]
Comment (stephendonner) [reply setup.cfg]: 8 or so occurrences in test/, according to the run in https://github.com/citp/OpenWPM/pull/200#issue-199336508.

File: requirements.txt (hunk 1 above)
Comment (stephendonner) [reply requirements.txt]: Oops, sorry; force of habit from the projects I work on.  We use pyup.io to auto-manage/update dependencies, and so we pin nearly everything.

I'll unpin.

**Pull request merged**